src/pages/home/report/ReportActionCompose.js

```diff
@@ -379,11 +384,12 @@ class ReportActionCompose extends React.Component {
      * @param {Boolean} shouldDebounceSaveComment
      */
     updateComment(newComment, shouldDebounceSaveComment) {
-        this.setState({
-            isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
-            value: newComment,
-        });
+        const isCommentEmpty = !!newComment.match(IS_EMPTY_PATTERN);
+        if (this.state.isCommentEmpty !== isCommentEmpty) {
+            this.setState({isCommentEmpty});
+        }
 
+        // TODO: when further deferring this code we can make the rendering more instantaneous
```

Comment:
Can we remove the TODO comment and just create an issue/PR for it please?

Reply:
moved to issue: https://github.com/Expensify/App/issues/11697